test/types/index.test-d.ts

```diff
@@ -1,4 +1,4 @@
-import wsPlugin, { SocketStream } from '../..';
+import * as wsPlugin from '../..';
```

Comment:
@Ethan-Arrowood @mfpopa @fox1t can you please explain this to me?

Reply:
I'll defer to @fox1t as he has a better understanding of this than I do

Reply:
These changes work for me.

I tested with `esModuleInterop` set to both false and true. Repository I used for testing this: https://github.com/mfpopa/fastify-faux-modules

Reply:
I checked out this branch and tested with both `esModuleInterop` set to true and false (it's false by default). I see find no issues.

Reply:
To clarify, when using `"esModuleInterop": false` (the default), import needs to be done like this `import * as wsPlugin from '../..';`. When using `"esModuleInterop": true`, import needs to be done like this `import wsPlugin from '../..';`.

Reply:
It works because export = syntax exports a so called namespace that is then imported by import *. Export default has to be used only in presence of .default prop on the exported object.